src/Modules/ModulesGraded.jl

```diff
@@ -2052,6 +2084,8 @@ end
 
 elem_type(::Type{FreeMod_dec{T}}) where {T} = FreeModElem_dec{T}
 parent_type(::Type{FreeModElem_dec{T}}) where {T} = FreeMod_dec{T}
+elem_type(::FreeMod_dec{T}) where {T} = FreeModElem_dec{T}
+parent_type(::FreeModElem_dec{T}) where {T} = FreeMod_dec{T}
```

Comment:
Sorry, but I became curious: Where do these types come from and where are they used? I thought that the graded modules are the same as the usual modules. Did anything change there?

Reply:
This is beyond my scope to be honest, maybe @jankoboehm knows.

Reply:
@HechtiDerLachs Please resolve this in a direct conversation with @jankoboehm